.storybook/theme.scss

```diff
@@ -1 +1,102 @@
-@use '../scss.scss';
+@use '../scss.scss' with (
+	$global: (
+		'blue-0': #EBEFFF,
+		'blue-1': #B0C3F2,
+		'blue-2': #7AA2E5,
+		'blue-3': #4A89D8,
+		'blue-4': #1E78CB,
+		'blue-5': #1458A9,
+		'blue-6': #0C3D87,
+		'blue-7': #062665,
+		'blue-8': #021443,
+		'blue-9': #000721,
+
+		'red-0': #FF8A8A,
+		'red-1': #E65555,
+		'red-2': #CE2929,
+		'red-3': #B50505,
+		'red-4': #9C0000,
+		'red-5': #840000,
+		'red-6': #6B0000,
+		'red-7': #520000,
+		'red-8': #3A0000,
+		'red-9': #210000,
+
+		'orange-0': #FA714B,
+		'orange-1': #E24F23,
+		'orange-2': #C93302,
+		'orange-3': #B12E00,
+		'orange-4': #982A00,
+		'orange-5': #802500,
+		'orange-6': #682000,
+		'orange-7': #4F1900,
+		'orange-8': #371200,
+		'orange-9': #1F0B00,
+
+		'green-0': #80D971,
+		'green-1': #57C244,
+		'green-2': #34AB1F,
+		'green-3': #189402,
+		'green-4': #137D00,
+		'green-5': #0F6600,
+		'green-6': #0C4F00,
+		'green-7': #083800,
+		'green-8': #052100,
+		'green-9': #020A00,
+
+		'neutral-0': #E6EFFF,
+		'neutral-1': #C3CFE3,
+		'neutral-2': #A3B0C6,
+		'neutral-3': #8593AA,
+		'neutral-4': #69778E,
+		'neutral-5': #4F5C71,
+		'neutral-6': #384355,
+		'neutral-7': #232B39,
+		'neutral-8': #10151C,
+		'neutral-9': #000000,
+
+		'spacing-0': 0.25em, // 4px
+		'spacing-1': 0.5em, // 8px
+		'spacing-2': 1em, // 16px
+		'spacing-3': 1.5em, // 24px
+		'spacing-4': 2em, // 32px
+		'spacing-5': 2.5em, // 40px
+		'spacing-6': 3em, // 48px
+		'spacing-7': 3.5em, // 56px
+		'spacing-8': 4em, // 64px
+		'spacing-9': 4.5em, // 72px
+		'spacing-10': 5em, // 80px
+		'spacing-11': 5.5em, // 88px
+		'spacing-12': 6em, // 96px
+		'spacing-13': 6.5em, // 104px
+		'spacing-14': 7em, // 112px
+		'spacing-15': 7.5em, // 120px
+
+		'font-weight-1': 100,
+		'font-weight-2': 200,
+		'font-weight-3': 300,
+		'font-weight-4': 400,
+		'font-weight-5': 500,
+		'font-weight-6': 600,
+		'font-weight-7': 700,
+		'font-weight-8': 800,
+		'font-weight-9': 900,
+
+		'font-size-0': 0.75rem, // 12px
+		'font-size-1': 0.875rem, // 14px
+		'font-size-2': 1rem, // 16px
+		'font-size-3': 1.125rem, // 18px
+		'font-size-4': 1.1875rem, // 19px
+		'font-size-5': 1.375rem, // 22px
+		'font-size-6': 1.5rem, // 24px
+		'font-size-7': 1.75rem, // 28px
+		'font-size-8': 2.25rem, // 36px
+
+		'line-height-0': 1rem, // 16px
```

Comment:
Should we use multiples for line height instead of units? Using `rem` spaces lines to the root em font size. Using a multiple will apply to the font in use.
Also like the spacing tokens, giving a nonzero value to a `-0` token may be confusing.
I'd recommend using hundreds for line height tokens, starting with 100.
```
line-height-100: 1;
```
I believe that would be an intuitive approach, as it could be equated with the percentage multiple.
```
line-height-150: 1.5;
line-height-200: 2;
```

Reply:
> as it could be equated with the percentage multiple

We should avoid equating a token name to be of some calculated value, as they will cause confusion